_scripts/instruction-widget/install.js

```diff
@@ -97,11 +97,12 @@ module.exports = function(context) {
     if (context.webserver == "apache") {
       context.package = "python-letsencrypt-apache";
     }
-    if (context.id == "debiantesting") {
+    if (context.os == "debianunstable") {
+      context.base_command = "certbot";
+      context.package = "python-certbot"
```

Comment:
nit: Based on [these descriptions](https://packages.debian.org/search?keywords=certbot&searchon=names&suite=unstable&section=all) I think we should tell people to install `certbot` (plus it's a shorter package name).